src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -2519,6 +2521,20 @@ class UserLDAPAuthSegment(BaseSegment):
     )
 
 
+@exasol_dialect.segment()
+class UserOPENIDAuthSegment(BaseSegment):
+    """user openid authentification."""
+
+    type = "openid_auth"
+    match_grammar = StartsWith(Sequence("BY", "OPENID"))
+    parse_grammar = Sequence(
```

Comment:
The `match_grammar` adds nothing here so suggest you merge the `match_grammar` and `parse_grammar` into one `match_grammar`.

Reply:
Thanks for the hint. done